ckan/controllers/group.py

```diff
@@ -295,15 +305,17 @@ def pager_url(q=None, page=None):
                     facets[facet] = facet
 
             # Facet titles
-            for plugin in plugins.PluginImplementations(plugins.IFacets):
-                if self.group_type == 'organization':
+            from ckan.controllers.organization import OrganizationController
+            if isinstance(self, OrganizationController):
```

Comment:
On the face of it these three are the same. I think I've been having half an eye on the situation where someone wants to inherit and customize OrganizationController. But really, rather than hypothecate wildly, I think we should just leave these alone until we have a proper use case and test, and then improve these. Otherwise my head will explode and will get it wrong anyway. What do you think?